openstack-tripleo-ui-deps.spec

```diff
@@ -12,15 +12,18 @@ URL:            http://tripleo.org
 # The source for this package was pulled from git.  Use the following commands
 # to generate the tarball.
 #
+# NOTE: Currently, this needs to be done on different boxes with
+#       each supported architecture.
 #   $ git clone https://github.com/openstack/tripleo-ui.git
 #   $ cd tripleo-ui
 #   $ git checkout %{commit}
 #   $ npm install
-#   $ tar czf tripleo-ui-deps-%{shortcommit}.tar.gz node_modules
-Source0:        tripleo-ui-deps-%{shortcommit}.tar.gz
+#   $ tar czf tripleo-ui-deps-%{shortcommit}-$(uname --hardware-platform).tar.gz node_modules
+Source0:        tripleo-ui-deps-%{shortcommit}-x86_64.tar.gz
+Source1:        tripleo-ui-deps-%{shortcommit}-ppc64le.tar.gz
 
 # Cannot build as noarch until nodejs is built from aarch64 in CBS
-ExclusiveArch: x86_64
+ExclusiveArch: x86_64 ppc64le
```

Comment:
as comment above says, extending list of exclusive arches is the wrong direction, it should be noarch eventually

Reply:
@honza I don't see favicons-webpack-plugin in npm-shrinkwrap.json is that dep gone now so phantonjs is no more and we can make ui-deps noarch RPM?!!

Reply:
Indeed, it was removed in https://review.openstack.org/534954 !

Reply:
Testing switch to noarch in 
https://github.com/rdo-common/openstack-tripleo-ui-deps/commit/b496c1c66df0a9b0f2da163b0906fe1c1b8bd037

Reply:
@honza  please test noarch ui-deps build http://cbs.centos.org/koji/buildinfo?buildID=22359